src/vit-testing/mainnet-tools/Cargo.toml

```diff
@@ -42,4 +42,4 @@ warp = "0.3.3"
 futures-util = "0.3.25"
 
 assert_fs = "1.0"
-prettytable-rs = "0.9.0"
+prettytable-rs = "0.10.0"
```

Comment:
Maybe move this dependency to the workspace level as it used in two different places